charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -63,6 +63,15 @@ defaultRules:
   # -- Additional labels for PrometheusRule alerts
   additionalRuleLabels: {}
 
+# Provide custom recording or alerting rules to be deployed into the cluster.
+additionalVictoriaMetricsMap:
```

Comment:
I think, that these lines must be commented as an example of usage.

Reply:
You are right. We can just set it like this:
additionalPrometheusRulesMap: {}